pkg/controller/core.oam.dev/v1alpha2/application/apply.go

```diff
@@ -223,7 +223,7 @@ func (h *AppHandler) collectHealthStatus(wl *appfile.Workload, appRev *v1beta1.A
 			isHealth = true
 		}
 		status.Message = configuration.Status.Apply.Message
-	} else {
+	} else if !skipStandardWorkload {
```

Comment:
please add an annotation to explain here

Reply:
added